gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/ConsultationListMapper.java

```diff
@@ -127,13 +129,16 @@ public ListResource mapToCategory(ListResource topic, RCMRMT030101UKCompoundStat
         return category;
     }
 
-    private DateTimeType getDate(RCMRMT030101UKCompoundStatement compoundStatement, ListResource parentList) {
+    private DateTimeType getDate(RCMRMT030101UKCompoundStatement compoundStatement, ListResource parentList,
+                                 RCMRMT030101UK04EhrExtract ehrExtract) {
         if (compoundStatement != null && compoundStatement.getAvailabilityTime() != null
             && compoundStatement.getAvailabilityTime().getValue() != null) {
             return DateFormatUtil.parseToDateTimeType(compoundStatement.getAvailabilityTime().getValue());
+        } else if (parentList.getDateElement() != null) {
+            return parentList.getDateElement();
+        } else {
+            return DateFormatUtil.parseToDateTimeType(ehrExtract.getAvailabilityTime().getValue());
```

Comment:
`EhrExtract` is different to the `EhrComposition`. The ticket specifies using the `EhrComposition / author` field.